src/views/authentication/Register.js

```diff
@@ -0,0 +1,206 @@
+import PropTypes from "prop-types";
+import React, { useState } from "react";
+import {
+  Alert,
+  View,
+  Button,
+  TextInput,
+  Text,
+  StyleSheet,
+  ActivityIndicator,
+} from "react-native";
+
+import { Formik } from "formik";
+import * as yup from "yup";
+import CTA from "../../components/authentication/CTA";
+import { Colors, Typography } from "../../constants/styles";
+import * as api from "../../api/Auth";
+import { AUTH_STYLES } from "../../constants/styles/auth";
+
+export default function Register(props) {
+  const { navigation } = props;
+
+  // 1 - DECLARE VARIABLES
+  const [setError] = useState(null);
```

Comment:
Wywala błąd setError is not a function. W tablicy musi być podana wartość i funkcja, która ma ją zmieniać.